db/seeds.rb

```diff
@@ -79,15 +79,54 @@ def position(type)
   surname = player['web_name']
   full_name = "#{player['first_name']} #{player['second_name']}"
   position = position(player['element_type'])
+
+   # Get specific player url
+   url = 'https://fantasy.premierleague.com/drf/element-summary/' << player['id'].to_s
+   response = HTTParty.get(url)
+ 
+    # Get specific player url
+    url = 'https://fantasy.premierleague.com/drf/element-summary/' << player['id'].to_s
+    response = HTTParty.get(url)
+  
+    # Gives hisory of the player
+    history_raw = JSON.parse(response.body)['history']
```

Comment:
shall we call this `player_history`, then we don't need the comment.

comments should always be 'why', not 'what'

Reply:
Updated